include/chopper/count/count_kmers.hpp

```diff
@@ -62,28 +55,33 @@ inline void count_kmers(robin_hood::unordered_map<std::string, std::vector<std::
     #pragma omp parallel for schedule(static) num_threads(config.num_threads)
     for (size_t i = 0; i < cluster_vector.size(); ++i)
     {
+        chopper::sketch::hyperloglog sketch(config.sketch_bits);
+
+        auto hash_seq_and_add_to_sketch = [&sketch] (auto & seq, auto && compute_fn)
+        {
+            for (auto && hash : seq | compute_fn)
+                sketch.add(reinterpret_cast<char*>(&hash), sizeof(hash));
+        };
```

Comment:
Can you move the definition out of the for-loop?

Reply:
Yes but then I need to take sketches as a parameter instead of capture